library/multipicker/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="im.vector.lib.multipicker">
 
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
This permission is already in the main manifest (https://github.com/vector-im/element-android/blob/main/vector/src/main/AndroidManifest.xml#L17) so I do not think it will change anything to add it in this module. The merged manifest will be the same.

For the changelog you could add a line like `Tentative fix of ...` or something similar.